public/pages/Dashboard/containers/Dashboard.js

```diff
@@ -288,20 +429,21 @@ export default class Dashboard extends Component {
         type: 'alertsDashboard',
         payload: {
           ...payload,
+          viewMode: this.state.viewMode,  // Pass viewMode to flyout
           openChainedAlertsFlyout: this.openChainedAlertsFlyout,
           closeFlyout: this.closeFlyout,
         },
       });
     }
   };
 
-  openChainedAlertsFlyout = (chainedAlert) => {
-    this.setState({ chainedAlert });
-  };
+  // openChainedAlertsFlyout = (chainedAlert) => {
```

Comment:
These sections that are commented out are needed for composite monitors aren't they?